src/scss/cdb-components/forms/sliders.scss

```diff
@@ -49,7 +119,16 @@
   content: '';
 }
 
-.CDB-Range::-webkit-slider-thumb::after {
+.CDB-Range::-ms-track {
+  height: $baseSize / 4;
+  margin: $baseSize 0;
+  border-radius: $baseSize / 2;
```

Comment:
Properties should be ordered height, margin, border, border-radius, background, color